packages/@react-native-windows/codegen/src/Cli.ts

```diff
@@ -173,7 +176,23 @@ function writeMapToFiles(map: Map<string, string>, outputDir: string) {
 
 function parseFlowFile(filename: string): SchemaType {
   try {
-    return parseFile(filename);
+    const schema = parseFile(filename);
+    // there will be at most one turbo module per file
+    const moduleName = Object.keys(schema.modules)[0];
+    if (moduleName) {
+      const spec = schema.modules[moduleName];
+      if (spec.type === 'NativeModule') {
+        const contents = fs.readFileSync(filename, 'utf8');
+        if (contents) {
+          if (contents.includes('TurboModuleRegistry.get<')) {
+            setOptionalTurboModule(spec, true);
```

Comment:
Can we include a comment that this is temporary until we get the new property added to the codegen schema.  Preferably with a link to a RN issue to add that property.

Reply:
added